eventually/vote/models.py

```diff
@@ -36,17 +41,31 @@ class Vote(models.Model):
     """
 
     MULTI_CHOICES = (
-        ('O', 'One'),
-        ('M', 'Multi'),
+        ("O", 'One'),
+        ("M", 'Multi'),
```

Comment:
what is the reason for such change `('O', 'One'),` to `("O", 'One'),` ?

Reply:
I delete that O and M and rename it to 0 and 1